src/pyuvdata/utils/uvcalibrate.py

```diff
@@ -417,5 +460,43 @@ def uvcalibrate(
         if uvcal_use.gain_scale is not None:
             uvdata.vis_units = uvcal_use.gain_scale
 
+    # Set pol convention properly
+    if uvcal_use.gain_scale is None:
+        # ignore all pol_convention stuff, because who knows???
+        warnings.warn(
+            "gain_scale is not set, so resulting uvdata will not have correct units"
```

Comment:
I think we need to add something to this warning message to make it clear that the `pol_convention` isn't being used in this case. Maybe we should also add something about this to the docstring under the `pol_convention` parameter.